app/react/Thesauri/ThesaurusCockpit.tsx

```diff
@@ -213,7 +244,8 @@ interface ThesaurusCockpitStore {
   thesauri: {
     thesaurus: IImmutable<ThesaurusSchema>;
     models: IImmutable<ClassifierModelSchema>[];
-    suggestions: IImmutable<SuggestionResultSchema>;
+    suggestionsTBPublished: IImmutable<SuggestionResultSchema>;
```

Comment:
Do you need the full SuggestionResultSchema? Or is a number enough?

Generally, I'm not a fan of adding too many simple fields to the reducers - how about thesauri/suggestionStats = { TBPublished: number, TBReviewed: number}?

Reply:
For TBPublished, that makes sense and I shall make it so.

For TBReviewed, we need the per-thesaurus-value counts as well so it cannot become just a number, unfortunately.

Reply:
Ah, I now read your comment differently than I did at first. You're suggesting that, in addition to the per-value suggestions to be reviewed (fka 'suggestions'), I add another one that is 'suggestionsStats' and just have the summary there?